examples/mps3-an536/src/bin/svc.rs

```diff
@@ -30,7 +30,7 @@ pub fn main() -> ! {
 }
 
 /// This is our SVC exception handler
-#[no_mangle]
+#[unsafe(no_mangle)]
```

Comment:
Is this OK with our MRSV?

Reply:
https://doc.rust-lang.org/edition-guide/rust-2024/unsafe-attributes.html

> To reflect the fact that these attributes can undermine Rust's safety guarantees, they are now considered "unsafe" and should be written as follows:

So it was added in Rust 1.82, and works in any edition. Edition 2024 just makes the new style mandatory.